pom.xml

```diff
@@ -141,6 +146,11 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.7</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>apache-httpcomponents-client-4-api</artifactId>
```

Comment:
Surely this belongs in `dependencyManagement` not `dependencies`?

Reply:
Done in f9010950dd71060542d8b4dd61267a17ab6342df.